packages/button/stories/button.stories.tsx

```diff
@@ -11,65 +11,57 @@ import { FaFacebook, FaTwitter } from "react-icons/fa"
 import { MdBuild, MdCall } from "react-icons/md"
 import { BeatLoader } from "react-spinners"
 import { motion } from "framer-motion"
+import { Meta, StoryFn } from "@storybook/react"
+import { ThemingProps } from "@chakra-ui/system"
+import { theme } from "@chakra-ui/theme"
+import { getThemingArgTypes } from "@chakra-ui/storybook-addon"
+import { pick } from "@chakra-ui/utils"
 import { Button, ButtonGroup, IconButton } from "../src"
 
 export default {
   title: "Components / Forms / Button",
   decorators: [
     (Story: any) => (
-      <Container mt="40px">
+      <Container mt="40px" display="flex" flexWrap="wrap" gap="4">
         <Story />
       </Container>
     ),
   ],
+} as Meta
+
+interface StoryProps extends ThemingProps<"Button"> {
+  children?: React.ReactNode
 }
 
-export const basic = () => (
-  <>
-    <Button colorScheme="gray">Button</Button>
-    <Button colorScheme="red">Button</Button>
-    <Button colorScheme="green">Button</Button>
-    <Button colorScheme="blue">Button</Button>
-    <Button colorScheme="teal">Button</Button>
-    <Button colorScheme="pink">Button</Button>
-    <Button colorScheme="purple">Button</Button>
-    <Button colorScheme="cyan">Button</Button>
-    <Button colorScheme="orange">Button</Button>
-    <Button colorScheme="yellow">Button</Button>
-  </>
-)
+export const basic: StoryFn<StoryProps> = (props) => <Button {...props} />
+basic.argTypes = {
+  ...getThemingArgTypes(theme, "Button"),
+  children: { type: "string" },
+}
```

Comment:
The new helper function allows us to show toggles for `variant`, `size` and `colorScheme` based on the given Chakra UI Theme.

Reply:
Love it!